core/unix/signal.c

```diff
@@ -3049,7 +3049,11 @@ mcontext_to_sigcontext(sig_full_cxt_t *sc_full, priv_mcontext_t *mc,
     if (TEST(DR_MC_CONTROL, flags)) {
         sc->SC_FIELD(arm_sp) = mc->r13;
         sc->SC_FIELD(arm_pc) = mc->r15;
-        sc->SC_FIELD(arm_cpsr) = mc->cpsr;
+        /* FIXME i#7207: The values of RES1 bits should probably be propagated
```

Comment:
Please add a line `Issue: #7207` to the PR description to get a link in the commit message.

Reply:
Done.